x-pack/plugins/security_solution/public/resolver/models/indexed_process_tree/isometric_taxi_layout.ts

```diff
@@ -219,10 +218,16 @@ function processEdgeLineSegments(
 
     const parentTime = eventModel.timestampSafeVersion(parent);
     const processTime = eventModel.timestampSafeVersion(process);
-    if (parentTime && processTime) {
-      edgeLineMetadata.elapsedTime = elapsedTime(parentTime, processTime) ?? undefined;
-    }
-    edgeLineMetadata.uniqueId = edgeLineID;
+
+    const timeBetweenParentAndNode =
+      parentTime !== undefined && processTime !== undefined
```

Comment:
:grey_question:  (parentTime && processTime && elapsedTime(parentTime, processTime)) maybe now that you've changed the return signature

Reply:
good catch

Reply:
that ends up w/ a type like `false | DurationDetails | undefined` so I'm going to skip it.